### PR TITLE
new example file

### PR DESCRIPTION
**Pull request by dshuvar**
I added new example file  which use functions rabbitmq_management and ssl certificates for rabbitMQ.

### CODE REVIEW

File: examples/rabbitmq_management_ssl.pp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+node 'rabbit.example.com' {
```
Comment (hunner) [inline examples/rabbitmq_management_ssl.pp]: I think either the node has to be taken out, or the rest of the code has to be indented to match.

File: examples/rabbitmq_management_ssl.pp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ssl_versions          => ['tlsv1.2', 'tlsv1.1'],
+  environment_variables => {
+    'NODENAME'    => 'rabbit',
+    'SERVICENAME' => 'Rabbit',
+    'CONFIG_FILE' =>  '/etc/rabbitmq/rabbitmq'
+  }
+
+}
+
+rabbitmq_plugin {'rabbitmq_management':
```
Comment (hunner) [inline examples/rabbitmq_management_ssl.pp]: Space after the open `{`

**Comment by wyardley** [comment]
Now that we have docs in puppet-strings, we should probably rework / review the examples, and maybe move the ones that are helpful into puppet-strings format. Let me know if you're interested in helping w/ docs work in general.

We should also probably just make `rabbitmq::server` private, and update the docs that declare it that way.

**Comment by wyardley** [comment]
I'm going to close this one for now, if there are additional examples, let's add them via Puppet strings, now that docs are auto-generated.

**Pull request closed without merging**